Gemfile

```diff
@@ -32,6 +32,8 @@ gem 'jquery-rails'
 
 # Use ActiveModel has_secure_password
 gem 'bcrypt', '~> 3.1.7'
+gem 'omniauth'
```

Comment:
You should always include a version restriction on a gem so you don't get major version upgrades unexpectedly.